Trying to build ROS nodes

Getting this error trying to build the ROS nodes. openvslam/ros/src/openvslam/src/run_localization.cc:71:14: error: ‘class openvslam::system’ has no member named ‘track_for_monocular’

--- a/ros/src/openvslam/src/run_localization.cc
+++ b/ros/src/openvslam/src/run_localization.cc
@@ -27,7 +27,6 @@
 #include <gperftools/profiler.h>
 #endif
 
-
 void mono_localization(const std::shared_ptr<openvslam::config>& cfg, const std::string& vocab_file_path,
                    const std::string& mask_img_path, const std::string& map_db_path, const bool mapping) {
     // load the mask image
@@ -68,7 +67,7 @@ void mono_localization(const std::shared_ptr<openvslam::config>& cfg, const std:
         const auto timestamp = std::chrono::duration_cast<std::chrono::duration<double>>(tp_1 - tp_0).count();
 
         // input the current frame and estimate the camera pose
-        SLAM.track_for_monocular(cv_bridge::toCvShare(msg, "bgr8")->image, timestamp, mask);
+        SLAM.feed_monocular_frame(cv_bridge::toCvShare(msg, "bgr8")->image, timestamp, mask);
 
         const auto tp_2 = std::chrono::steady_clock::now();
 

--- a/ros/src/openvslam/src/run_slam.cc
+++ b/ros/src/openvslam/src/run_slam.cc
@@ -58,7 +58,7 @@ void mono_tracking(const std::shared_ptr<openvslam::config>& cfg, const std::str
         const auto timestamp = std::chrono::duration_cast<std::chrono::duration<double>>(tp_1 - tp_0).count();
 
         // input the current frame and estimate the camera pose
-        SLAM.track_for_monocular(cv_bridge::toCvShare(msg, "bgr8")->image, timestamp, mask);
+        SLAM.feed_monocular_frame(cv_bridge::toCvShare(msg, "bgr8")->image, timestamp, mask);
 
         const auto tp_2 = std::chrono::steady_clock::now();
 

